sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/ChaosFaultResource.cs

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.ResourceManager.CosmosDB
+{
+    /// <summary>
+    /// A Class representing a ChaosFaultResource along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier"/> you can construct a <see cref="ChaosFaultResource"/>
+    /// from an instance of <see cref="ArmClient"/> using the GetChaosFaultResource method.
+    /// Otherwise you can get one from its parent resource <see cref="CosmosDBAccountResource"/> using the GetChaosFaultResource method.
+    /// </summary>
+    public partial class ChaosFaultResource : ArmResource
```

Comment:
Better to add a prefix since the name is quite generic.

Reply:
Added